flink-connectors/flink-connector-hive/pom.xml

```diff
@@ -1099,33 +1007,8 @@ under the License.
 			<properties>
 				<hive.version>3.1.3</hive.version>
 				<derby.version>10.14.1.0</derby.version>
-				<!-- need a hadoop version that fixes HADOOP-14683 -->
-				<hivemetastore.hadoop.version>2.8.2</hivemetastore.hadoop.version>
```

Comment:
If we're removing all references to this property then we should also remove it from the root pom.

Reply:
I think they were all resolved in a later commit already